django/views/defaults.py

```diff
@@ -59,12 +74,19 @@ def server_error(request, template_name='500.html'):
     try:
         template = loader.get_template(template_name)
     except TemplateDoesNotExist:
+        if template_name != ERROR_500_TEMPLATE_NAME:
+            warnings.warn(
+                "A template_name has been provided, but the template cannot be found. "
```

Comment:
I think it would be best if these warnings at least named the relevant view which received a bad template name (and perhaps the template name which could not be found, as well). The information can be found in the stack trace (at least the former can, not sure about the latter), but there's no reason not to make it available right in the error message, is there?

Same for the other errors.